server/py/services/api/crud/runtimes/nuclio/function.py

```diff
@@ -362,15 +362,20 @@ def _apply_escaped_config(config, parent_key, items: dict):
         mlrun.utils.update_in(config, f"{parent_key}.\\{key}\\", value)
 
 
-def _resolve_env_vars(function, auth_info=None):
+def _enrich_config_spec(function, username):
```

Comment:
why not passing auth_info entirely? a better practice as you might late on use its other fields